_build/stac.hbs

```diff
@@ -16,7 +17,7 @@
   "keywords": [
 {{{listJson Tags 4}}}
   ],
-  "license": "{{{escapeJson LicenseType}}}",
+  "license": {{#if LicenseType }} "{{{LicenseType}}}"{{else}} "{{escapeJson License}}"{{/if ~}},

```

Comment:
Should it be like this rather: {{#if LicenseType }} "{{{**escapeJson** LicenseType}}}"{{else}} "{{escapeJson License}}"{{/if ~}}, with escapeJson?

Reply:
It shouldn't matter as the LicenseType value is always "proprietary" for now but it's certainly safer to escape it --> added